frontend/src/components/helpers/custom-tools/CustomToolsHelper.js

```diff
@@ -81,6 +81,16 @@ function CustomToolsHelper() {
       .then((res) => {
         const data = res?.data;
         updatedCusTool["llmProfiles"] = data;
+
```

Comment:
@tahierhussain @harini-venkataraman Any reason why this file has extension of `.js`. If it returns JSX element, it is better to keep it `.jsx`

Reply:
I remember @tahierhussain confirmed this to be .js, since it has pure logic. However I will let him give more inputs.

Reply:
@Deepak-Kesavan This is a helper component that contains mostly the logic. The component is wrapped around the prompt studio components. I decided to keep it's extension as `.js` becuase almost the entire file consists of logic. We're just returning the `<Outlet />` component.

Reply:
@Deepak-Kesavan This is more of a helper component that will wrap around the prompt studio components. I decided to use a `.js` extension because almost the entire component consists of logic. We're only returning the `<Outlet />`, which will render the components wrapped inside this component.

Reply:
Since `<Outlet />` is a jsx component as well as we're returning spinner which is also a jsx component it will be more appropriate to name it as jsx.